packages/next-dashboard/src/subscriptionProviders/SubscriptionPoller.js

```diff
@@ -65,18 +66,24 @@ export default class SubscribtionPoller<Data: {} = {}> extends EventEmitter
     }
   }
 
-  addFetcher(fetcher: PollingFetcher | PollingFetcher[]) {
-    const fetchers: PollingFetcher[] = Array.isArray(fetcher)
+  addFetcher(fetcher: PollingFetcher<Data> | PollingFetcher<Data>[]) {
+    const fetchers: $ReadOnlyArray<PollingFetcher<Data>> = Array.isArray(
+      fetcher,
+    )
       ? fetcher
       : [fetcher];
+    // Not sure how to type this. Technically flow is correct I suppose?
+    // Since keys COULD be symbols, but 1. Flow doesn't support symbols
+    // 2. using symbols as keys in this instance would be silly.
+    // $FlowIssue
     this.fetchers = this.fetchers.concat(fetchers);
```

Comment:
What is the type error? What is flow correct about? What do you want to type how?